waitress/pantry/viewsets.py

```diff
@@ -1,16 +1,13 @@
 from datetime import date
 
-import pytz
-from django.shortcuts import get_object_or_404
-from drf_yasg import openapi
-from drf_yasg.utils import swagger_auto_schema
 from rest_framework import status as status_code
 from rest_framework import viewsets
 from rest_framework.decorators import action
 from rest_framework.response import Response
 
-from app.decorators import guard
-from app.models import Pantry, SlackUser, Passphrase, MealSession
+from app.models import SlackUser, Passphrase
+
+from pantry.models import Pantry
```

Comment:
There's one last bit of this, I didn't make clear. I sincerely apologize for that.

We need to move the `viewset` for Pantry into the `pantry` app too, so concerns are totally separated.
After which you can then import the `PantryViewset` from the pantry app into the [urls.py](https://github.com/andela/waitress/blob/master/waitress/urls.py#L23) file.

Reply:
Mistake.